11/SUMMARY.md

```diff
@@ -1,2 +1,2 @@
-How can we create more natural looking textures like surface of the roads, rocks, trees and clouds? Noise function is the answer.
-Since Ken Perlin invented his first noise algorithm in 80s, the technique has been extensively used throughout computer graphics and simulations. Even if you have never heard of the name, it's not possible you have never seen it. Let's look step by step at how the function is built and works. We also cover more efficient version of the algorithm called simplex noise.
+How can we create more natural looking textures like the surface of roads, rocks, trees and clouds? Noise functions are the answer.
+Since Ken Perlin invented his first noise algorithm in 80's, the technique has been extensively used throughout computer graphics and simulations. Even if you have never heard of the name, it's unlikely you have not seen it. Let's take a step by step look at how the function is built and works. We also cover a more efficient version of the algorithm called simplex noise.
```

Comment:
The actual year that Perlin noise was invented is 1983. Anything less than a year looks like a placeholder.

If the invention(1983)-then-publishing(1985) span makes the use case too... wait for it... noisy, then [see the english stack exchange on the use of apostrophes in the mentioning of decades](https://english.stackexchange.com/a/299512). I'd personally prefer to see "the early-1980s"

Reply:
cool! thanks